spec/helpers/claims_helper_spec.rb

```diff
@@ -177,4 +178,28 @@ def current_user
       end
     end
   end
+
+  describe '#fee_shared_headings' do
+    subject(:headings) { fee_shared_headings(claim, 'test.scope') }
+
+    let(:claim) { build :claim }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4011/files#r652792036)